packages/test-runner-chrome/src/ChromeLauncher.ts

```diff
@@ -1,15 +1,11 @@
 import * as puppeteerCore from 'puppeteer-core';
-import {
-  Browser,
-  Page,
-  LaunchOptions,
-  launch as puppeteerCoreLaunch,
-  BrowserContext,
-} from 'puppeteer-core';
+import { Browser, Page, launch as puppeteerCoreLaunch, BrowserContext } from 'puppeteer-core';
 import { BrowserLauncher, TestRunnerCoreConfig } from '@web/test-runner-core';
 import { findExecutablePath } from './findExecutablePath';
 import { ChromeLauncherPage } from './ChromeLauncherPage';
 
+export type PuppeteerLaunchOptions = Parameters<typeof puppeteerCoreLaunch>[0];
```

Comment:
This will not be needed in future after puppeteer/puppeteer#6888 is released with [PuppeteerNodeLaunchOptions](https://github.com/puppeteer/puppeteer/commit/98c81458c27f378eb66c38e1620e79e2ffde418e#diff-0f963dc502fa6fef5a2bbec28bd0959ff22bfe8bb6ca1c1c97353a5b6bf29de6R120-R122).

Reply:
Updated to `8.0.0` so now the correct type is used.